src/components/dialog/dialog.scss

```diff
@@ -37,8 +37,11 @@ $dialog-sizes: (
   box-shadow: 0 10px 20px rgba($grey-shadow, 0.2);
   position: fixed;
   top: 50%;
-  z-index: $z-dialog;
-
+  min-width: 664px;
+  max-width: 800px;
+  width: calc(50%);
+  height: calc(80%);
```

Comment:
what was the reason for adding these?

Reply:
The dialog div wasn't narrowing. There was nothing with a width above it in the html hierarchy.

Reply:
This appears to break the width/height of dialogs.

Reply:
Try the `size` prop.

Reply:
I think I must have 'fiddled' with the size prop by accident and wondering why the width had change. I think we do however need the height. This might have already been an issue before when the dialog content height was bigger than the screen height